package.json

```diff
@@ -43,5 +40,12 @@
   },
   "engines": {
     "node": ">=6.1.0"
```

Comment:
Do we need to update this line to 12 if we're updating to 12 elsewhere. Sure 12 is greater than 6, but given how many versions there's in between, it might be nice to bump this up here as well.

Reply:
YES! Thank you 🌟